docs/installation.rst

```diff
@@ -8,33 +8,44 @@ Installation and configuration
 Stable release
 --------------
 
-To install ``django-import-export-extensions``, run this command in your terminal:
+To install ``django-import-export-extensions``, run the following command in your terminal:
 
-.. code-block:: console
+Using pip:
+
+.. code-block:: shell
 
     $ pip install django-import-export-extensions
 
-This is the preferred method to install ``django-import-export-extensions``, as it will always install the most recent stable release.
+Using uv:
+
+.. code-block:: shell
 
-If you don't have `pip`_ installed, this `Python installation guide`_ can guide
-you through the process.
+    $ uv pip install django-import-export-extensions
+
+Using poetry:
+
+.. code-block:: shell
 
-.. _pip: https://pip.pypa.io
-.. _Python installation guide: http://docs.python-guide.org/en/latest/starting/installation/
+    $ poetry add django-import-export-extensions
```

Comment:
If you deleted `$` sign in `contributing.rst` so maybe you should delete it here too?
```suggestion
    poetry add django-import-export-extensions
```